pkg/trie/triedb/codec/key_test.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package codec
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/mock/gomock"
+)
+
+func repeatBytes(n int, b byte) (slice []byte) {
```

Comment:
nit/maybe reverse order of args so number is last. Feels a bit more intuitive to me, but ultimately up to you here

Reply:
Done! thks